Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst

```diff
@@ -0,0 +1,2 @@
+Show string value of :opcode:`CONTAINS_OP` oparg in :mod:`dis` output.
+Patch by [Alexandr153].
```

Comment:
```suggestion
Patch by Alexandr153.
```

I'm sorry that my suggestion template was misunderstood :')

Reply:
That's okay, this is my first experience with CPython, thank you

Reply:
Don't worry, you're doing great for a first experience!